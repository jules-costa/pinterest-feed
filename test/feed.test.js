import React from 'react';
import ReactDOM from 'react-dom';
import Feed from '../frontend/feed';
import renderer from 'react-test-renderer';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';

it('should render an item ', () => {
  const tree = toJson(shallow(<Feed items={[]} />));
  expect(tree).toMatchSnapshot();
});

it('should match its snapshot with items', () => {
  const items = ['cute cat', 'adorable', 'I love!'];
  const tree = renderer.create(
    <Feed items={items} />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
