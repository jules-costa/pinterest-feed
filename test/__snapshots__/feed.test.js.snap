// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match its snapshot with items 1`] = `
<div
  className="masonry"
>
  <div
    className="item"
  >
    <a
      href={[Function]}
    >
      <img
        src={undefined}
      />
    </a>
    <h5 />
  </div>
  <div
    className="item"
  >
    <a
      href={[Function]}
    >
      <img
        src={undefined}
      />
    </a>
    <h5 />
  </div>
  <div
    className="item"
  >
    <a
      href={[Function]}
    >
      <img
        src={undefined}
      />
    </a>
    <h5 />
  </div>
</div>
`;

exports[`should render an item  1`] = `
<div
  className="masonry"
/>
`;
